### PR TITLE
add support for v-model & fix tabs initial width bug & improve docs

### PR DESCRIPTION
**Pull request by baiyaaaaa**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by Leopoldthecoder** [comment]
这个要合到 dev 上？

**Comment by Kingwl** [comment]
终于。。。

**Comment by baiyaaaaa** [comment]
@Leopoldthecoder 对，没有 breaking change ，只是增加了v-model 的用法

File: examples/docs/en-US/tabs.md [hunk 1]
```diff
@@ -84,7 +102,7 @@ Tabs styled as cards.
 
 Closable tabs.
 
-:::demo You can set `closable` attribute in `el-tabs`. It accept `Boolean` and Tab will be closable when the boolean is `true`.
+:::demo You can set `closable` attribute in `el-tabs` to make the tabs to be closable, `closable` is also can be setting in the tab panel to make partial tabs to be closable.
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/tabs.md]: You can set the `closable` attribute in `el-tabs` to make all tabs closable. Also, `closable` can be set in a tab panel to make that specific tab closable.

File: examples/docs/en-US/tabs.md [hunk 2]
```diff
@@ -27,11 +38,11 @@ Divide data collections which are related yet belong to different types.
 
 Basic and concise tabs.
 
-:::demo Tabs provide a selective card functionality and it can be achieved by just using `el-tabs` and child element `el-tab-pane`. In these two elements, we provide a list of attributes. The `label` in `el-tab-pane` determines the label of selective cards, and you can write content in the label. In this example, we add a `active-name` attribute indicating the active card in `el-tabs`, which can take a `String` value. In the `el-tab-pane` you can set corresponding `name` attribute, and if there is no `name`, the default sequence is `1`/`2`/`3`/`4`. In this example, the selected card is card 2. If `name` is omitted, setting `active-name` to `2` can reach the same goal.
+:::demo Tabs provide a selective card functionality.It will select the first card by default and you can set the `value` attribute to indicating the active card as you want.
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/tabs.md]: Tabs provide a selective card functionality. By default the first tab is selected as active, and you can activate any tab by setting the `value` attribute.

**Comment by baiyaaaaa** [comment]
#1610

**Pull request merged**